From david-b@pacbell.net  Mon Jan 26 08:46:59 2009
From: David Brownell <david-b@pacbell.net>
Date: Mon, 26 Jan 2009 02:05:43 -0800
Subject: USB: musb: Kconfig fix
To: Greg KH <greg@kroah.com>
Cc: Bryan Wu <cooloney@kernel.org>, Felipe Balbi <felipe.balbi@nokia.com>
Message-ID: <200901260205.44131.david-b@pacbell.net>


From: David Brownell <dbrownell@users.sourceforge.net>

The Blackfin MUSB Kconfig text didn't properly parenthesise its
dependencies.  This was visible in non-Blackfin configs by the
way the user interfaces lost track of dependencies, when doing
a bunch of test builds.

Signed-off-by: David Brownell <dbrownell@users.sourceforge.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/musb/Kconfig |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/usb/musb/Kconfig
+++ b/drivers/usb/musb/Kconfig
@@ -50,7 +50,7 @@ comment "OMAP 343x high speed USB suppor
 	depends on USB_MUSB_HDRC && ARCH_OMAP34XX
 
 comment "Blackfin high speed USB Support"
-	depends on USB_MUSB_HDRC && (BF54x && !BF544) || (BF52x && !BF522 && !BF523)
+	depends on USB_MUSB_HDRC && ((BF54x && !BF544) || (BF52x && !BF522 && !BF523))
 
 config USB_TUSB6010
 	boolean "TUSB 6010 support"
